src/java/org/apache/cassandra/index/sai/disk/v1/IndexWriterConfig.java

```diff
@@ -328,6 +361,32 @@ else if (options.get(MAXIMUM_NODE_CONNECTIONS) != null ||
                 similarityFunction = sourceModel.defaultSimilarityFunction;
             }
 
+            if (options.containsKey(VECTOR_FLUSH_THRESHOLD))
+            {
+                try
+                {
+                    vectorFlushThreshold = Integer.parseInt(options.get(VECTOR_FLUSH_THRESHOLD));
```

Comment:
I think we want to put a lower bound on this setting. Not sure if it needs to be a guardrail or an environment variable, but I would guess we wouldn't want to accept something less than 100k, for example.

Reply:
now it's not user-facing, we don't need guardrails